### DISCUSSION TITLE
How to get stream fps

### DISCUSSION THREAD

**Discussion by CursingThomas** [Motion-Project/motion#1740] [Q&A]
I have noticed that the fps text in the bottom left only changes on motion event, so does the movie_fps parameter in the struct. I was wondering if i can get these variables to update realtime, instead of like they are currently behaving.

**Comment by tosiara** [comment]
I'm sorry, I didn't get the question. Can you provide more details, logs, screenshots, etc?

**Reply by CursingThomas** [reply]
hi, sorry if i was vague. 

I would like to get the fps of the client side stream in real time. Is this possible or already integrated?

**Reply by tosiara** [reply]
yes, it is available as `%{fps}` modifier

**Reply by CursingThomas** [reply]
Hi, yes i gathered from the documentation :) Though it does not seem to work, i am clearly achieving stuttering and slower framerate in my client stream. Though the fps stays at 30. Any ideas as to why? Or how to measure delay on the stream in a different manner?
